libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -659,30 +668,40 @@ namespace SohImGui {
                 EnhancementSliderInt("King Zora Speed: %dx", "##WEEPSPEED", "gMweepSpeed", 1, 5, "");
 
                 EnhancementCheckbox("Skip Text", "gSkipText");
+                Tooltip("Holding down B skips text");
                 EnhancementCheckbox("Minimal UI", "gMinimalUI");
+                Tooltip("Hides most of the UI when not needed");
                 EnhancementCheckbox("MM Bunny Hood", "gMMBunnyHood");
+                Tooltip("Wearing the Bunny Hood grants a speed increase like in Majora's Mask");
                 EnhancementCheckbox("Visual Stone of Agony", "gVisualAgony");
+                Tooltip("Displays an icon and plays a sound when Stone of Agony should be activated, for those without rumble");
 
                 ImGui::Text("Graphics");
                 ImGui::Separator();
 
                 EnhancementCheckbox("N64 Mode", "gN64Mode");
+                Tooltip("Sets aspect ratio to 4:3 and lowers resolution to 240p, the N64's native resolution");
 
                 EnhancementCheckbox("Animated Link in Pause Menu", "gPauseLiveLink");
                 EnhancementCheckbox("Enable 3D Dropped items", "gNewDrops");
                 EnhancementCheckbox("Faster Block Push", "gFasterBlockPush");
                 EnhancementCheckbox("Dynamic Wallet Icon", "gDynamicWalletIcon");
+                Tooltip("Changes the rupee in the wallet icon to match the wallet size you currently have");
                 EnhancementCheckbox("Always show dungeon entrances", "gAlwaysShowDungeonMinimapIcon");
+                Tooltip("Alays shows entrances to dungeons on the minimap");
 
                 ImGui::Text("Fixes");
                 ImGui::Separator();
                 EnhancementCheckbox("Fix L&R Pause menu", "gUniformLR");
+                Tooltip("Makes the L and R buttons in the pause menu the same color");
                 EnhancementCheckbox("Fix Dungeon entrances", "gFixDungeonMinimapIcon");
+                Tooltip("Hides the dungeon entrance icon in the top left of the screen");
```

Comment:
This also fix the Y position of the icon once you completed a dungeon, without it you will never see that icon it correct position even if you have completed a dungeon.

I think it should say something like :
``Show dungeon entrances icon only when it should be.``